sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/ctl/AsyncCtlWorkload.java

```diff
@@ -370,4 +375,13 @@ private void initializeReporter(Configuration configuration) {
                 .build();
         }
     }
+
+    protected Mono sparsityMono() {
+        Duration duration = configuration.getSparsityWaitTime();
+        if (duration != null && !duration.isZero()) {
```

Comment:
Why no skip warmup operation like in regular bench mark ? Also can we make is a static method as one place and user across places ?

Reply:
Yea agree that is a good point.

So after comparing AsyncBenchmark and AsyncCtlWorkload, I see there are a lot common methods, like creating cosmosClient, create database, initialize reporter etc. And since they all part of the benchmark, so I think it makes sense to have a base class to concentralize all those common methods. 

So I created a BenchmarkBase class instead. And moved some methods to the base class including the sparsityMono and initializeMetersIfSkippedEnoughOperations. 

@moderakh @simplynaveen20 @kushagraThapar  Can u take a look again since there are some refactor since last time, thanks~ (and let me know if I have gone toooo far...)